lang/po/Firmware_nl.po

```diff
@@ -0,0 +1,1787 @@
+#: ultralcd.cpp:3484
+msgid "Calibrating XYZ. Rotate the knob to move the Z carriage up to the end stoppers. Click when done."
+msgstr "Kalibreren van XYZ. Draai de knop om de Z-wagen omhoog te gaan tot het einde stoppers. Druk knop als klaar."
+
+# MSG_CALIBRATE_Z_AUTO c=20 r=2
+#: messages.c:20
+msgid "Calibrating Z"
+msgstr "Kalibrere Z"
+
+# MSG_MOVE_CARRIAGE_TO_THE_TOP_Z c=20 r=8
+#: ultralcd.cpp:3484
+msgid "Calibrating Z. Rotate the knob to move the Z carriage up to the end stoppers. Click when done."
+msgstr "Kalibreren van Z. Draai de knop om de Z-wagen omhoog te gaan tot het einde stoppers. Druk knop als klaar."
+
+# MSG_HOMEYZ_DONE
+#: ultralcd.cpp:821
+msgid "Calibration done"
+msgstr "Kalibratie klaar"
+
+# MSG_MENU_CALIBRATION
+#: messages.c:61
+msgid "Calibration"
+msgstr "Kalibratie"
+
+# 
+#: ultralcd.cpp:4828
+msgid "Cancel"
+msgstr "Annuleren"
+
+# MSG_SD_REMOVED
+#: ultralcd.cpp:8812
+msgid "Card removed"
+msgstr "SD verwijderd"
+
+# MSG_NOT_COLOR
+#: ultralcd.cpp:2719
+msgid "Color not correct"
+msgstr "Kleur niet juist"
+
+# MSG_COOLDOWN
+#: messages.c:23
+msgid "Cooldown"
+msgstr "Afkoelen"
+
+# 
+#: ultralcd.cpp:4584
+msgid "Copy selected language?"
+msgstr "Geselecteerde taal kopieren?"
+
+# MSG_CRASHDETECT_ON
+#: messages.c:27
+msgid "Crash det.   [on]"
+msgstr "Crashdet.   [aan]"
+
+# MSG_CRASHDETECT_NA
+#: messages.c:25
+msgid "Crash det.  [N/A]"
+msgstr "Crashdet.   [N/V]"
+
+# MSG_CRASHDETECT_OFF
+#: messages.c:26
+msgid "Crash det.  [off]"
+msgstr "Crashdet.   [uit]"
+
+# MSG_CRASH_DETECTED c=20 r=1
+#: messages.c:24
+msgid "Crash detected."
+msgstr "Crash gedetecteerd."
+
+# 
+#: Marlin_main.cpp:596
+msgid "Crash detected. Resume print?"
+msgstr "Crash gedetecteerd. Print voorzetten?"
+
+# 
+#: ultralcd.cpp:1837
+msgid "Crash"
+msgstr ""
+
+# MSG_CURRENT c=19 r=1
+#: ultralcd.cpp:5979
+msgid "Current"
+msgstr "Actueel"
+
+# MSG_DATE c=17 r=1
+#: ultralcd.cpp:2197
+msgid "Date:"
+msgstr "Datum:"
+
+# MSG_DISABLE_STEPPERS
+#: ultralcd.cpp:5721
+msgid "Disable steppers"
+msgstr "Motoren uit"
+
+# MSG_BABYSTEP_Z_NOT_SET c=20 r=12
+#: messages.c:14
+msgid "Distance between tip of the nozzle and the bed surface has not been set yet. Please follow the manual, chapter First steps, section First layer calibration."
+msgstr "Afstand tussen tip van de tuit en het print oppervlak is nog niet vastgesteld. Volg de handleiding, First steps, sectie First layer calibration."
+
+# MSG_WIZARD_REPEAT_V2_CAL c=20 r=7
+#: ultralcd.cpp:5134
+msgid "Do you want to repeat last step to readjust distance between nozzle and heatbed?"
+msgstr "Wilt u de laatste stap herhalen om de afstand tussen de tuit en de bed opnieuw in te stellen?"
+
+# MSG_EXTRUDER_CORRECTION c=10
+#: ultralcd.cpp:5201
+msgid "E-correct:"
+msgstr "E-correctie:"
+
+# MSG_EJECT_FILAMENT c=17 r=1
+#: messages.c:53
+msgid "Eject filament"
+msgstr "Fil. uitwerpen"
+
+# MSG_EJECTING_FILAMENT c=20 r=1
+#: mmu.cpp:1413
+msgid "Ejecting filament"
+msgstr "Werp filament uit"
```

Comment:
Translation seems closer to "remove filament" than ejecting, if there's space I'd suggest
"Fil. aan het uitwerpen" ("filament is ejecting")

Reply:
Is te lang met c=22 r=1

Reply:
"Fil. word ontladen"?